_posts/2019-01-23-new-in-2019-unlimited-tutoring-for-all-students.md

```diff
@@ -13,6 +13,8 @@ excerpt: At Austin Coding Academy, we take pride in providing our students an
   level of support and community. One element of student support that we're most proud
   of having available in 2019 is unlimited tutoring for all ACA students.
 img: "/assets/images/coding-tutoring-session.jpg"
+width: 6720
+height: 4480
```

Comment:
@char-cole what are these for?

Reply:
Most of the blog images are in a standard aspect ratio, but for the few that aren't, these fields will let amp-image resize them appropriately instead of stretching them. It may increase the vertical size of a post on the index page, but it doesn't disrupt the rows and columns.